pythran/tests/test_numpy_random.py

```diff
@@ -633,3 +633,810 @@ def numpy_random_bytes1(n):
+
+    def test_numpy_negative_binomial0a(self):
+        """ Check negative_binomial with 1 argument with mean and variance. """
+        code = """
+        def numpy_negative_binomial0a(size):
+            from numpy.random import negative_binomial
+            from numpy import var, mean
+            n = 1
+            p = 1
+            rmean = (n*(1-p))/p
+            rvar = (n*(1-p))/p**2
+            a = [negative_binomial(n,p) for x in range(size)]
+            return (abs(mean(a)- rmean) < .05 and abs(var(a) - rvar) < .05)
+        """
+        self.run_test(code, 10 ** 6, numpy_negative_binomial0a=[int])
+
+    def test_numpy_negative_binomial0b(self):
+        """ Check negative_binomial with 2 argument with mean and variance. """
+        code = """
+        def numpy_negative_binomial0b(size):
+            from numpy.random import negative_binomial
+            from numpy import var, mean, sqrt
+            n = 1
+            p = 1
+            rmean = (n*(1-p))/p
+            rvar = (n*(1-p))/p**2
+            a = negative_binomial(n, p, size)
+            print(mean(a))
+            return (abs(mean(a)- rmean) < 0.05 and abs(var(a) - rvar) < .05)
+        """
+        self.run_test(code, 10 ** 6, numpy_negative_binomial0b=[int])
+
+    def test_numpy_negative_binomial2(self):
+        """Check negative_binomial with shape argument with mean and variance."""
+        code = """
+        def numpy_negative_binomial2(size):
+            from numpy.random import negative_binomial
+            from numpy import mean, var
+            n = 1
+            p = 1
+            rmean = (n*(1-p))/p
+            rvar = (n*(1-p))/p**2
+            a = negative_binomial(n, p , size=(size, size))
+            return (abs(mean(a)- rmean) < .05 and abs(var(a) - rvar) < .05)
+        """
+        self.run_test(code, 10 ** 3, numpy_negative_binomial2=[int])
+
+    ###########################################################################
+    #Tests for numpy.random.standard_exponential
+    ###########################################################################
+
+    def test_numpy_standard_exponential0(self):
+        """ Check standard_exponential without argument with mean and variance. """
+        code = """
+        def numpy_standard_exponential0(size):
+            from numpy.random import standard_exponential
+            from numpy import var, mean
+            a = [standard_exponential() for x in range(size)]
+            return (abs(mean(a) - 1) < .05 and abs(var(a) - 1) < .05)
+        """
+        self.run_test(code, 10 ** 5, numpy_standard_exponential0=[int])
+
+    def test_numpy_standard_exponential1(self):
+        """ Check standard_exponential with size argument with mean and variance."""
+        code = """
+        def numpy_standard_exponential1(size):
+            from numpy.random import standard_exponential
+            from numpy import var, mean
+            a = standard_exponential(size)
+            print(mean(a))
```

Comment:
please remove these print in the test :-)

Reply:
Yes sorry, I will remove them in the next commit, I forgot to do it :)